package.json

```diff
@@ -116,5 +116,8 @@
     "relay-local-schema": "^0.8.0",
     "relay-runtime": "^5.0.0",
     "rimraf": "^3.0.0"
+  },
+  "resolutions": {
+    "relay-compiler/core-js": "2.6.9"
```

Comment:
this forces the package to install even if relay-compiler doesn't specify it?

Reply:
I'm not sure. Maybe? But it's forcing Relay Compiler to use this, rather than the hoisted v2, anyway.